src/config.c

```diff
@@ -450,6 +462,19 @@ void loadServerConfigFromString(char *config) {
             }
         }
 
+        //If there's no matching above, we try matching them with deprecated configs
```

Comment:
one of the very few consistent styling guides in redis, is that it avoids using line comments.
```suggestion
        /* If there's no matching above, we try matching them with deprecated configs */
```